.github/workflows/e2e_learner_flow_skill_editor_and_embedding.yml

```diff
@@ -23,6 +23,8 @@ jobs:
           architecture: 'x64'
       - name: Setup python by installing wheel
         run: pip install wheel==0.35.0
+      - name: Merge from Base Branch if Running on a PR
+        uses: ./.github/actions/merge-base-branch
```

Comment:
Do we need the name property here? will it use the name defined in the action itself?

Reply:
Any specific reason for doing it after installing the wheel?

Reply:
Yes, I think it will use the action name if we leave out `name`. It doesn't matter whether we do this before or after installing wheel. I put the step here because this is right before we start referencing the code

Reply:
done